docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+.. post:: October 9, 2017
+   :tags: newsletter
+
+
+#####################################
+Write the Docs Newsletter – October
+#####################################
+
+Hello friends and fellow documentarians!
+
+Kelly O. here, happy to be arriving in your inbox for another round-up of docs-wisdom gleaned from our community. It's been a big month, since last we spoke. We had our fourth European conference, we've surpassed 2500 members in Slack, AND it's officially been a year since we started this here newsletter experiment. Happy newsletterversary!
+
+We're going into the last few months of this year with our sights set on shoring up some of our community frameworks, as well as developing some new projects (secret! exciting! forthcoming!). We're so happy to have you all along for the ride!
+
+Unsurprisingly, we had a bountiful harvest of interesting and useful conversations to choose from for this issue. We've tried to assemble a good mix here for your perusal. Have a great month!
+
+*Errata: In assembling the meetup calendar below, I suspect I did something wrong last month, to have come up with so few. If you ran a Write the Docs meetup last month that I didn't catch in the newsletter, I'm so sorry!*
+
+*************************************************
+Putting our URLs to work – for us and our readers
+*************************************************
+How do we make our URLs meaningful to our readers? What should we do when content inevitably moves (or is removed)? How can we make sure all of the great content we write remains discoverable?
+
+When discussing these questions this month, the community started from a point of agreement: good URL paths should carry meaning. For example, a lot of doc sites have URLs that reflect their content hierarchy, like ``site/category/article-title``. Ideally, in such situations, if a reader truncates the path to ``site/category/`` they should find a list of every article in the category.But content restructuring and obsolescence are facts of life for most documentarians. It's unrealistic to think that `Cool URIs don't change <https://www.w3.org/Provider/Style/URI>`_. When the inevitable does happen, it's important to make it as easy as possible for readers to find new locations. That means redirects!
+
+Creating redirects can be difficult to manage when there are lots of external links when lots of URLs are changing. Sadly, most documentation tools – with the notable exception of MindTouch – don't offer automated redirects. One suggested approach was to use analytics to prioritize your most critical URLs, setting up manual redirects for those pages first to keep your content discoverable. For long term URL hygiene, folks suggested automated link-checking tools to find track down and update broken links and stray pages.
```

Comment:
"...when there are lots of external links when lots of URLs are changing." - Do you want to delete the "when lots of URLs are changing" part?

Reply:
ah, just needed an 'or'